src/proto/faabric.proto

```diff
@@ -10,6 +10,9 @@ service FunctionRPCService {
     rpc ShareFunction (Message) returns (FunctionStatusResponse) {
     }
 
+    rpc Flush (Message) returns (FunctionStatusResponse) {
+    }
```

Comment:
These methods have to take a request and return a response type, even though in this case both are ignored.